students/WesleyM/Lesson9/mailroom.py

```diff
@@ -0,0 +1,120 @@
+class donor:
+    def __init__(self, name, donation = None):
+        if not donation:
+            donation = []
+        self.name = name
+        self.donation = donation
```

Comment:
This assignment should go in an else clause above, or could be done as a one-liner:
```
self.donation = donation if donation is not None else []
```